### ISSUE TITLE
MATT and Better Roll Tables

### ISSUE THREAD

**Issue by WinterrrWarlock** [ironmonk88/monks-active-tiles#756]
Honestly MATT is a total game changer, but is there any way to have the "Roll Table" option default to BRT if it's installed?

I have a journal set up with 10 different pages that my Adventure's Guild uses as "job requests". If I open the roll table and roll it as Roll + (which means BRT handles it) then it rolls, selects a random job, outputs it to chat and my players can see that exact page in the journal. If i use the stock roll table, then it just opens the journal to the last page that was opened regardless of what is rolled.

I have a trigger setup that when players click the Job Request board, it automatically rolls the roll table and outputs it to chat. The downside is that it uses the stock roll table option and thus they keep pulling the same job over and over. 

TLDR: being able to use the Roll Table function and have it work with BRT would be awesome.

**Comment by Crowguard** [comment]
Duplicate of #582

**Comment by Anbu0002** [comment]
it might be a duplicate but i hope it helps to get his attention

**Comment by Crowguard** [comment]
@Anbu0002 If anything, its more likely to be the wrong kind of attention that just irritates developers. Posting duplicates wastes more developer time, because they have to deal with closing/responding/updating both tickets.
